pkg/plugins/runtime/gateway/filter_chain_generator.go

```diff
@@ -281,11 +282,6 @@ func newFilterChain(ctx xds_context.Context, info *GatewayResourceInfo) *envoy_l
 		),
 	)
 
-	// TODO(jpeach) add compressor filter.
-	// TODO(jpeach) add decompressor filter.
-	// TODO(jpeach) add grpc_web filter.
-	// TODO(jpeach) add grpc_stats filter.
```

Comment:
Are the others not needed? I'm thinking we don't need decompressor since we can just leave data compressed.

Reply:
AFAICT, decompressor isn't something that's generally useful. The best rationale I can come up with is a client that PUTs compressed content to a backend that can't accept it. I expect that's a less common case.

As for gRPC, I think we should have some more thought about what makes an excellent gRPC experience and decide the most appropriate set of filters on that basis.